java/webrtc-hello-world/src/main/java/com/bandwidth/webrtc/examples/helloworld/HelloWorldController.java

```diff
@@ -130,7 +123,7 @@ public HelloWorldController(AccountProperties accountProperties, VoiceProperties
         calls.put(callId, response);
 
         // Generate transfer BXML from the device token and return that to the Voice API
-        String transferBxml = generateTransferBxml(response.getToken());
+        String transferBxml = WebRtcTransfer.generateBxml(response.getToken());
```

Comment:
Reminder to update this to use generateBxmlVerb() as well as update pom.xml with new Java SDK version